src/gateways/mappers/IBeaconRequest.ts

```diff
@@ -0,0 +1,19 @@
+export interface IBeaconRequest {
+  data: {
+    type: string;
+    id: string;
+    attributes: {
+      hexId?: string;
+      status?: string;
+      type?: string;
+      manufacturer?: string;
+      createdDate?: string;
+      model?: string;
+      manufacturerSerialNumber?: string;
+      chkCode?: string;
+      protocolCode?: string;
+      batteryExpiryDate?: string;
+      lastServicedDate?: string;
+    };
+  };
+}
```

Comment:
`relationships` key not included, waiting until we get to updating relationships between a beacon and its owner and uses etc. before implementing.